v3/plugins/redhat/quarkus/1.2.0/meta.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+publisher: redhat
+name: quarkus
```

Comment:
I would name the plugin `quarkus-java8` because it is bringing quarkus ext + java and java8 runtime. Would make sence to avoid confusion like `do I need to install java8 or java11 plugin if I already have quarkus ?

Reply:
I've been assuming that for CRW 2.1 we'd need a JDK11 runtime for use with quarkus.

Are you saying that quarkus works with BOTH JDK8 and 11, and therefore we will need two sidecar containers, one per JDK flavour?

Reply:
I've updated `name` and `displayName`